tests/models/qwen2_5/colqwen2_5/test_modeling_colqwen2_5.py

```diff
@@ -0,0 +1,134 @@
+import logging
+from typing import Generator, cast
+
+import pytest
+import torch
+from datasets import load_dataset
+from PIL import Image
+from transformers.utils.import_utils import is_flash_attn_2_available
+
+from colpali_engine.models import ColQwen2_5, ColQwen2_5_Processor
+from colpali_engine.utils.torch_utils import get_torch_device, tear_down_torch
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope="module")
+def model_name() -> str:
+    return "vidore/colqwen2.5-v0.2"
```

Comment:
If you agree with what I told you on Slack:
- Rename `vidore/colqwen2.5-v0.2` → `vidore/colqwen2.5-v1.0`
- Rename the model name used in the tests.

```suggestion
    return "vidore/colqwen2.5-v1.0"
```